README.md

```diff
@@ -93,7 +93,7 @@ We encourage users to openly license their manuscripts, which is the default as
 [![License: CC0 1.0](https://img.shields.io/badge/License%20Parts-CC0%201.0-lightgrey.svg)](https://creativecommons.org/publicdomain/zero/1.0/)
 
 Except when noted otherwise, the entirety of this repository is licensed under a CC BY 4.0 License ([`LICENSE.md`](LICENSE.md)), which allows reuse with attribution.
-Please attribute by linking to https://github.com/trang1618/tpot-ds-ms.
```

Comment:
You will want to keep this line as is.

Reply:
I see. I thought this part refers to the Manubot tool itself. Fixed [here](https://github.com/trang1618/tpot-ds-ms/pull/8/commits/fcb528c8a16dfb53e6e79ccefdf1ecccaccfb0c9).